### PR TITLE
Editor dynamic type support

### PR DESCRIPTION
**Pull request by joewalsh**

### CODE REVIEW

File: Wikipedia/assets/codemirror/codemirror-index.html [hunk 1]
```diff
@@ -388,8 +388,9 @@
       })
     }
     
-    const setupCodemirror = (language, direction, themeName, callback) => {
+    const setupCodemirror = (language, direction, themeName, textSizeAdjustment, callback) => {
       applyTheme(themeName)
+      document.querySelector('body').style['font-size'] = (textSizeAdjustment/100.0) + 'em'
```
[the remark this answers is not in the document]
Comment (joewalsh) [reply Wikipedia/assets/codemirror/codemirror-index.html]: Tried that initially but it didn't work. The line height increased but the font size did not.

[the remark this answers is not in the document]
Comment (montehurd) [reply Wikipedia/assets/codemirror/codemirror-index.html]: huh that's curious. no worries.

File: Wikipedia/assets/codemirror/resources/addon/search/search.js [hunk 2]
```diff
@@ -137,20 +137,14 @@
         }
       }
 
-      const focusedMatchID = `cm-searching-focus-id-${focusedMatchIndex}`;
```
Comment (montehurd) [inline Wikipedia/assets/codemirror/resources/addon/search/search.js]: oh were the removals from this file deliberate? they seem unrelated to text size...

Comment (joewalsh) [reply Wikipedia/assets/codemirror/resources/addon/search/search.js]: They were changes related to find in page that were brought into www but not built into the assets folder
